FAQ.md

```diff
@@ -49,6 +49,19 @@ However, this will change in the future as js-ipfs gets GC and we want to make s
 
 All databases (feeds) are shared between peers, so nobody "owns them" like users do in ssb (afaik). Multiple peers can append to the same db. @tyleryasaka is right in that each peer has their own copy of the db (the log) and they may have different versions between them but, as @tyleryasaka (correctly) describes, through the syncing/replication process the peers exchange "their knowledge of the db" (heads) with each other, the dbs/logs get merged. This is what the "CRDT" in ipfs-log enables. But from address/authority/ownership perspective, they all share the same feed.
 
+### I'm getting a lot of 429 (Too Many Requests) errors when I run OrbitDB
+
+This happens when there's only one node with data available, and the system isn't able to effectively get all of the data it needs from it. in order to get around this, instantiate IPFS with preload enabled:
+
+```js
+this.ipfs = new Ipfs({
+  preload: { enabled: false },
+  // ...
+}
+```
+
+This will preload data on the remote nodes, spreading out the demand across multiple nodes to avoid hitting one node too much.
```

Comment:
This is correct but we're actual _disabling_ preload in this instance because the preload nodes are sometimes down.

Reply:
I should have actually read that. Edited and pushed change.